docker-compose.yml

```diff
@@ -36,7 +37,8 @@ services:
       - data-broker
     environment:
       FALCON_SETTINGS_MODULE: 'history.settings.docker'
-      DOJOT_SERVICE_MANAGEMENT: 'internal'
+      DOJOT_MANAGEMENT_USER: 'persister'
```

Comment:
For now, only python library separates `management_tenant` from `management_user`